examples/Learning_example/learning_example.ipynb

```diff
@@ -0,0 +1,2295 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    class Node:
</pre><p>So, I am wondering here how much of this class we should show and how much we can put in a file and forget. Maybe it would make sense to put this thing into a node.py?</p><p> </p><p>Overall, could you add docstrings for the methods? :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aidotse/PASEOS/pull/107/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>